src/main/java/Bots/LocaleMiddleman.java

```diff
@@ -0,0 +1,45 @@
+package Bots;
+
+import static Bots.Main.guildLocales;
+
+// maybe this could become more than just a workaround for createQuickError and toTimestamp, but who knows.
+public class LocaleMiddleman {
+    private static final ThreadLocal<Long> threadedGuildID = new ThreadLocal<>();
```

Comment:
I'm not overly thrilled with the idea of relying on threading magic to determine what language should be used (blueprint success and error replies should probably be repurposed into a method of CommandEvent for logical flow reasons), but functionally I'm pretty sure it's at least valid, so it's not the biggest concern, though something we can and should refactor

Reply:
your suggestion of making create quick error and create quick success a method of CommandEvent is good, I dont think there is an edge case where it would be required without a guild but if there is then there is a solution for that which i will implement if i find (managerLocalise use)

Reply:
I have now addressed this, LocaleMiddleman is no more, blueprint success and error messages are now a part of CommandEvent (needed to overload for cases where an event does not exist)

It wasn't mentioned here but timeunits were also threaded, they are now not, the code for that got moved to LocaleManager.